packages/create-redwood-app/template/api/jest.config.js

```diff
@@ -1 +1,6 @@
-module.exports = require('@redwoodjs/testing/config/jest/api')
+const config = {
```

Comment:
I suggest we add a link to Jest Project config section in redwoodjs.com/docs/testing section [TODO]

Reply:
That testing doc is _huge_... I think we should probably have a separate section. Maybe we can repurpose https://redwoodjs.com/docs/babel to be "App Configuration: Build, Dev, Test"  or something similar
![image](https://user-images.githubusercontent.com/1521877/151707645-1b020292-2a91-4316-aa25-9f8acb4e2aea.png)

Reply:
Oh, that's a good idea to add a new config doc specific to Build, Dev, Test, etc. At first, I was thinking to add to the existing "App Configuration: ...", but that's already long. So agreed a new one is preferred. (And we can link from other docs, easy peasy).

My primary concern for docs like this is similar to "App Configuration: redwood.toml" — making them findable by an assortment of keywords. What if we tried something like this?
- Nav Title: "App Configuration: Babel, Webpack, and Jest"
- Page h1 Title: "Config and Settings: Build, Dev, and Test"

fwiw

@jtoar (Just brainstorming.) What we really need is an entire Docs section for Configuration that includes a page for everthing config related, including links to the files in crwa/template. Another thing we could do is audit + improve the comments within config files; e.g. `./graphql.config.js` probably doesn't need a Doc but some explanation at the top of the file sure seems helpful.

Reply:
Yeah totally what I was going for, a place to document all the various configs we have so far I can think of:
- jest.config.js
- storybook.config.js or storbook manager (I don't know how this works)
- api/server.config.js
- webpack.config.js
- codegen.yml (I totally forgot to document this!)
- babel (?)
- graphql.config.js
- eslint? 

I think this particular file being long is not gonna be a problem, because people would most likely come to this section looking for something specific. But broadly speaking these are "all the other configs" in a redwood project. I think its important to find a good name, but I like the idea of colocating it

Reply:
In progress. Resolving conversation.